index.html

```diff
@@ -8366,6 +8367,42 @@ <h2>Pointer actions</h2>
 
 </ol>
 
+<p>When required to <dfn>dispatch a pointerWheel action</dfn> with
+ arguments <var>source id</var>, <var>action object</var>,
+ <var>input state</var> and <var>tick duration</var> a
+ <a>remote end</a> must run the following steps:
+
+<ol>
+ <li><p>Let <var>delta x</var> be equal to the <code>x</code>
+  property of <var>action object</var>.
+
+ <li><p>Let <var>delta y</var> be equal to the <code>y</code>
+  property of <var>action object</var>.
+
+ <li><p>Let <var>delta z</var> be equal to the <code>z</code>
+  property of <var>action object</var>.
+
+ <li><p>Let <var>mode</var> be equal to the <code>mode</code>
+  property of <var>action object</var>.
+
+ <li><p><a data-lt=perform-actions>Perform implementation-specific 
+  action dispatch steps</a> equivalent to spinning a wheel with
+  vector magnitude (<var>delta x</var>, <var>delta y</var>, 
+  <var>delta z</var>) for one tick on the pointer of ID
+  <var>source id</var> having type <var>pointerType</var> at
+  viewport x coordinate <var>x</var>, viewport y
+  coordinate <var>y</var>, with buttons <var>buttons</var> depressed,
+  in accordance with the requirements of [[!UI-EVENTS]] and
+  [[!POINTER-EVENTS]].  The generated events must
+  set <code>ctrlKey</code>, <code>shiftKey</code>, <code>altKey</code>,
+  and <code>metaKey</code> from the <a>calculated global key state</a>.
+  Type specific properties for the pointer that are not 
+  exposed through the webdriver API must be set to the default value
```

Comment:
I know this is copied, but an s/webdriver/WebDriver/ here would be good. We can fix up the other instances of this in a follow-up.